helm_deploy/prison-staff-hub/templates/secrets.yaml

```diff
@@ -16,3 +16,5 @@ data:
   NOTIFY_API_KEY: {{ .Values.secrets.NOTIFY_API_KEY | b64enc | quote }}
   WANDSWORTH_OMU_EMAIL: {{ .Values.secrets.WANDSWORTH_OMU_EMAIL | b64enc | quote }}
   WANDSWORTH_VLB_EMAIL: {{ .Values.secrets.WANDSWORTH_VLB_EMAIL | b64enc | quote }}
+  CONTENTFUL_SPACE_ID: {{ .Values.secrets.CONTENTFUL_SPACE_ID | b64enc | quote }}
+  CONTENTFUL_ACCESS_TOKEN: {{ .Values.secrets.CONTENTFUL_ACCESS_TOKEN | b64enc | quote }}
```

Comment:
I assume these have been added to secrets manager? Otherwise the deploys to any env will fail.

Reply:
Aye they're in.